src/server.ts

```diff
@@ -31,19 +35,33 @@ if (process.env.NODE_ENV === 'production') {
 
 getFilesWithKeyword('router', __dirname + '/app').forEach((file: string) => {
   const { router } = require(file);
-  app.use(router);
+
+  router ? app.use(router) : console.warn(`${file} doesn't seen to export a 'router'`);
 });
 
 /************************************************************************************
  *                               Express Error Handling
  ***********************************************************************************/
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-app.use((err: Error, req: express.Request, res: express.Response, next: express.NextFunction) => {
+app.use((error: any, req: express.Request, res: express.Response, next: express.NextFunction) => {
+  if (error.response) {
+    if (error.response.status === 409) {
+      return res.status(400).send({
+        message: 'Conflict updating the information',
+      });
+    }
+  } else if (error.request) {
+    // `error.request` is an instance of XMLHttpRequest in the browser and an instance of
+    // http.ClientRequest in node.js
+    console.log(error.request);
+  } else {
+    console.log('Error', error.message);
+  }
+
   return res.status(500).json({
-    errorName: err.name,
-    message: err.message,
-    stack: err.stack || 'no stack defined',
+    errorName: error.name,
+    message: error.message,
+    ...(error.stack && { stack: error.stack }),
```

Comment:
Let's get started with good stuff:

This is a global error handling fallback. As long as we implement middlewares correctly, this puppy should provide a safe and clear error response to help troubleshoot, etc.